packages/web/package.json

```diff
@@ -53,6 +53,8 @@
     "@osmosis-labs/stores": "^5.1.0",
     "@osmosis-labs/types": "^1.0.0",
     "@osmosis-labs/utils": "^1.0.0",
+    "@radix-ui/react-checkbox": "^1.0.4",
+    "@radix-ui/react-icons": "^1.3.0",
```

Comment:
https://www.npmjs.com/package/@radix-ui/react-icons

We should be careful with bundle size. What is this used for?